roles/kubernetes-apps/csi_driver/gcp_pd/defaults/main.yml

```diff
@@ -1,3 +1,14 @@
 ---
 gcp_pd_csi_controller_replicas: 1
-gcp_pd_csi_driver_image_tag: "v0.7.0-gke.0"
+# gcp_pd_csi_driver_image_tag: "v1.4.0"
+
+# csi_attacher_image_repo: "{{ kube_image_repo }}/sig-storage/csi-attacher"
```

Comment:
Why don't you enable these lines?

Reply:
Hello @oomichi ,
sorry for late reply. This is commented, because they are defined in [roles/download/defaults/main.yml](https://github.com/kubernetes-sigs/kubespray/pull/8616/files/13c8c1435b48a62aaa0c12ac9700b0d06d885fa1#diff-008da745794ecb794ca7727d7b637510bd1ef58da8837c4d4feaa95e3bcc5eca) by default and not needed rewrite  in this area but I kept them for possible rewrite. Of course, this can be deleted because it's copied from mensioned file.

Reply:
@oomichi Hello, ok I deleted them.